dds_web/api/project.py

```diff
@@ -188,6 +188,115 @@ def post(self):
 
         return {"message": return_message}
 
+    @auth.login_required(role=["Unit Admin", "Unit Personnel"])
+    @logging_bind_request
+    @json_required
+    @handle_validation_errors
+    def patch(self):
```

Comment:
We don't have this on the other `ProjectStatus` endpoints but we should add the `handle_db_error` decorator so that the endpoint returns a `DatabaseError` instead of pure SQLAlchemy errors etc in the case of a database issue. E.g. if there's an error in the lines leading up to the `try` block. 
```suggestion
    @auth.login_required(role=["Unit Admin", "Unit Personnel"])
    @logging_bind_request
    @json_required
    @handle_validation_errors
    @handle_db_error
    def patch(self):
```